src/vaadin-contextmenu-items-mixin.html

```diff
@@ -0,0 +1,241 @@
+<link rel="import" href="../../vaadin-item/src/vaadin-item.html">
+<link rel="import" href="../../vaadin-list-box/src/vaadin-list-box.html">
+
+<script>
+  /**
+   * @namespace Vaadin.ContextMenu
+   */
+  window.Vaadin = window.Vaadin || {};
+  window.Vaadin.ContextMenu = window.Vaadin.ContextMenu || {};
+
+  /**
+   * @polymerMixin
+   */
+  Vaadin.ContextMenu.ItemsMixin = superClass => class ItemsMixin extends superClass {
+
+    static get properties() {
+      return {
+        /**
+          * @typedef MenuItem
```

Comment:
BTW incorrect comment indentation, should be like this:

```
/**
 * @typedef MenuItem
 * ...
 */
```

Reply:
Done.